### PR TITLE
added more info and examples about data ingestion and collectors to VictoriaLogs docs

### PR DESCRIPTION
**Pull request by Amper**
…

### CODE REVIEW

**Review by tenmozes — APPROVED** [review]

File: docs/VictoriaLogs/Roadmap.md [hunk 1]
```diff
@@ -17,8 +17,6 @@ The following functionality is planned in the future versions of VictoriaLogs:
 
 - Support for [data ingestion](https://docs.victoriametrics.com/VictoriaLogs/data-ingestion/) from popular log collectors and formats:
   - Promtail (aka Grafana Loki)
-  - Vector.dev
-  - Fluentbit
```
Comment (zekker6) [inline docs/VictoriaLogs/Roadmap.md:21]: should this be removed?

File: docs/VictoriaLogs/data-ingestion/Logstash.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Logstash setup
 
+[Logstash](https://www.elastic.co/guide/en/logstash/8.8/introduction.html) log collector supports
+[Elasticsearch output](https://www.elastic.co/guide/en/logstash/current/plugins-outputs-elasticsearch.html) compatible with
```
Comment (zekker6) [inline docs/VictoriaLogs/data-ingestion/Logstash.md:4]: Logstash uses opensearch plugin for output, maybe we should use reference to this plugin?
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/deployment/docker/victorialogs/logstash/pipeline.conf

Here is a readme for it which describes why opensearch plugin is used - https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/deployment/docker/victorialogs/logstash/README.md

File: docs/VictoriaLogs/Roadmap.md (hunk 1 above)
Comment (Amper) [reply docs/VictoriaLogs/Roadmap.md:21]: Yes, i've removed it from roadmap, because we already support it

Comment (zekker6) [reply docs/VictoriaLogs/Roadmap.md:21]: ahh, sorry, missed that it is roadmap

File: docs/VictoriaLogs/data-ingestion/Logstash.md (hunk 2 above)
Comment (Amper) [reply docs/VictoriaLogs/data-ingestion/Logstash.md]: fixed

File: docs/VictoriaLogs/data-ingestion/Fluentbit.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+     host localhost
+     port 9428
+     uri /insert/jsonline/?_stream_fields=stream&_msg_field=log&_time_field=date
+     format json_lines
+     json_date_format iso8601
+     header AccountID 12
+     header ProjectID 23
+```
+
+More info about output tuning you can find in [these docs](https://www.elastic.co/guide/en/beats/filebeat/current/elasticsearch-output.html).
```
[the remark this answers is not in the document]
Comment (Amper) [reply docs/VictoriaLogs/data-ingestion/Fluentbit.md]: fixed

**Review by zekker6 — APPROVED** [review]

**Pull request merged**